### ISSUE TITLE
SCT raises false error event not finding start of Scylla Server running on Scylla Cloud

### ISSUE THREAD

**Issue by vponomaryov** [scylladb/scylla-cluster-tests#4209]
### Prerequisites

* [x] Are you rebased to master ?
* [x] Is it reproducible ?
* [x] Did you perform a cursory search if this issue isn't opened ?

### Versions
* **SCT**: master
* **scylla**: any

### Logs
* test_id: aa248904-49f6-412f-bbf7-9ddb0e18c4e0
* job log: [[e.x. https://cloudius-jenkins-test.s3.amazonaws.com/5a2c3b72-40f8-4992-8b5a-5c4aec8bf0c6/job-2019-05-14T09.59-c85338a.zip]](https://jenkins.scylladb.com/view/siren/job/siren-tests/job/longevity-tests/job/sct-siren-compatibility-checker/53)

### Description

Running latest SCT with SCylla cloud (sirenada) we get following error event that marks whole run as Failed:
```
2021-12-10 18:05:39.414: (TestFrameworkEvent Severity.ERROR) period_type=one-time event_id=9f1fc3fb-723d-4224-b59c-39fb626f0f35, source=ScyllaServerStatusEvent message=Did not find any events of type <class 'sdcm.sct_events.database.ScyllaServerStatusEvent'> with period type begin, event data: {}
```

### Steps to Reproduce

1. Run SCT with Scylla Cloud
2. See error

**Expected behavior:** Scylla Server Start event must be caught correctly.

**Actual behavior:** Scylla Server Start event is not caught by SCT.

**Comment by dkropachev** [comment]
Thanks Guys, addressed here - https://github.com/scylladb/scylla-cluster-tests/pull/4196

**Comment by ShlomiBalalis** [comment]
@dkropachev Please make sure the fix will be backported to 4.6 as well
